sympy/solvers/pde.py

```diff
@@ -898,7 +898,7 @@ def pde_separate_add(eq, fun, sep):
     Helper function for searching additive separable solutions.
 
     Consider an equation of two independent variables x, y and a dependent
-    variable w, we look for the product of two functions depending on different
+    variable w, we look for the sum of two functions depending on different
     arguments:
 
     `w(x, y, z) = X(x) + y(y, z)`
```

Comment:
Having reread the documentation for `pde_separate_add` and compared with that for its sibling `pde_separate_mul` and their parent `pde_separate`, I found more inconsistencies (in particular between the text of the help-string and the examples of code in it) and so propose instead of this a new pull request #11302.